pykern/pkcli/ci.py

```diff
@@ -31,14 +31,18 @@ def check_prints():
     from pykern import pkcli
 
     res = []
-    for f in pkio.walk_tree(pkio.py_path(), _FILE_TYPE):
+    p = pkio.py_path()
+    for f in pkio.walk_tree(p, _FILE_TYPE):
+        f = "./" + p.bestrelpath(f)
         if re.search(_EXCLUDE_FILES, str(f)):
```

Comment:
There are two primary cases:

1. We are running from top of repo. This is what "ci" is supposed to do.
2. Ad hoc testing, iwc arguments are passed (currently not supported)

I'm focusing on case (1). We can assert we are running from the top of the repo (there should be a setup.py in the pwd, for example). Given that, the regex can make more assumptions:

```py
    f"^{test.SUITE_D}/.*(?:{pkunit.DATA_DIR_SUFFIX}|{pkunit.WORK_DIR_SUFFIX})/"
    + f"|^\\w+/{pksetup.PACKAGE_DATA}/"
    + r"|pkdebug[^/]*\.py$"
    + r"|(?:^|/)\."
```

In the ad hoc case (2) we can just accept file(s) as arguments and check prints on those. I don't think we need to support that case right now, though.

Also "f" is  a string already so "str()" is unnecessary.